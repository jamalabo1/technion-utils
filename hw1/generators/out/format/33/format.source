D W 201 1
B Y 676 2
B Y 927 3
D Y 936 4
A Y 566 5
B Z 687 6
D Z 278 7
A Y 671 8
C Y 300 9
A Z 756 10
C Z 37 11
D Y 195 12
B X 92 13
A Z 83 14
D Y 702 15
B W 2 16
A X 467 17
A Y 723 18
B W 770 19
C Z 400 20
C Y 346 21
B W 672 22
D Z 517 23
B Y 634 24
D X 618 25
D Z 259 26
B Y 926 27
D Y 623 28
A Z 935 29
C W 349 30
D W 728 31
B W 68 32
A Y 812 33
A X 405 34
A X 881 35
D Y 621 36
B Z 869 37
B X 959 38
D X 950 39
B W 365 40
A W 663 41
B X 560 42
D X 36 43
C W 956 44
D W 479 45
A Y 943 46
B W 137 47
A Y 43 48
B W 287 49
A X 473 50
A Y 306 51
A Y 263 52
A Z 834 53
D Y 205 54
C Y 774 55
B Z 868 56
D Z 530 57
D Z 793 58
A W 1000 59
B Z 979 60
C X 598 61
A Y 621 62
D X 197 63
C Z 718 64
B X 730 65
A X 653 66
A W 989 67
A X 184 68
A Z 402 69
D W 774 70
C Z 214 71
A Y 848 72
C X 148 73
C W 504 74
A Y 810 75
A W 771 76
B W 908 77
D Y 212 78
B Z 206 79
D Z 788 80
B X 913 81
B W 567 82
A X 936 83
B X 641 84
D Z 3 85
A Y 124 86
C Y 453 87
B X 335 88
A W 912 89
B Z 850 90
C X 291 91
A Z 204 92
D X 98 93
C Y 67 94
C Y 895 95
C Y 452 96
B Z 677 97
C W 545 98
B Z 287 99
A W 309 100
D Z 817 101
C Y 686 102
D Y 736 103
D W 187 104
A Z 741 105
B W 394 106
B W 308 107
B X 443 108
D Y 90 109
D X 278 110
C W 279 111
C X 989 112
B W 403 113
B W 468 114
B X 156 115
B Y 652 116
A X 585 117
C W 526 118
A Y 97 119
A X 617 120
A Z 95 121
D Y 853 122
A W 844 123
A Z 912 124
A W 911 125
D X 243 126
B Z 323 127
A X 331 128
C X 146 129