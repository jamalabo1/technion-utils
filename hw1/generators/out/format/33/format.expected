Blockchain Info:
1
Sender Name: C
Receiver Name: X
Transaction Value: 146
Transaction timestamp: 129
2
Sender Name: A
Receiver Name: X
Transaction Value: 331
Transaction timestamp: 128
3
Sender Name: B
Receiver Name: Z
Transaction Value: 323
Transaction timestamp: 127
4
Sender Name: D
Receiver Name: X
Transaction Value: 243
Transaction timestamp: 126
5
Sender Name: A
Receiver Name: W
Transaction Value: 911
Transaction timestamp: 125
6
Sender Name: A
Receiver Name: Z
Transaction Value: 912
Transaction timestamp: 124
7
Sender Name: A
Receiver Name: W
Transaction Value: 844
Transaction timestamp: 123
8
Sender Name: D
Receiver Name: Y
Transaction Value: 853
Transaction timestamp: 122
9
Sender Name: A
Receiver Name: Z
Transaction Value: 95
Transaction timestamp: 121
10
Sender Name: A
Receiver Name: X
Transaction Value: 617
Transaction timestamp: 120
11
Sender Name: A
Receiver Name: Y
Transaction Value: 97
Transaction timestamp: 119
12
Sender Name: C
Receiver Name: W
Transaction Value: 526
Transaction timestamp: 118
13
Sender Name: A
Receiver Name: X
Transaction Value: 585
Transaction timestamp: 117
14
Sender Name: B
Receiver Name: Y
Transaction Value: 652
Transaction timestamp: 116
15
Sender Name: B
Receiver Name: X
Transaction Value: 156
Transaction timestamp: 115
16
Sender Name: B
Receiver Name: W
Transaction Value: 468
Transaction timestamp: 114
17
Sender Name: B
Receiver Name: W
Transaction Value: 403
Transaction timestamp: 113
18
Sender Name: C
Receiver Name: X
Transaction Value: 989
Transaction timestamp: 112
19
Sender Name: C
Receiver Name: W
Transaction Value: 279
Transaction timestamp: 111
20
Sender Name: D
Receiver Name: X
Transaction Value: 278
Transaction timestamp: 110
21
Sender Name: D
Receiver Name: Y
Transaction Value: 90
Transaction timestamp: 109
22
Sender Name: B
Receiver Name: X
Transaction Value: 443
Transaction timestamp: 108
23
Sender Name: B
Receiver Name: W
Transaction Value: 308
Transaction timestamp: 107
24
Sender Name: B
Receiver Name: W
Transaction Value: 394
Transaction timestamp: 106
25
Sender Name: A
Receiver Name: Z
Transaction Value: 741
Transaction timestamp: 105
26
Sender Name: D
Receiver Name: W
Transaction Value: 187
Transaction timestamp: 104
27
Sender Name: D
Receiver Name: Y
Transaction Value: 736
Transaction timestamp: 103
28
Sender Name: C
Receiver Name: Y
Transaction Value: 686
Transaction timestamp: 102
29
Sender Name: D
Receiver Name: Z
Transaction Value: 817
Transaction timestamp: 101
30
Sender Name: A
Receiver Name: W
Transaction Value: 309
Transaction timestamp: 100
31
Sender Name: B
Receiver Name: Z
Transaction Value: 287
Transaction timestamp: 99
32
Sender Name: C
Receiver Name: W
Transaction Value: 545
Transaction timestamp: 98
33
Sender Name: B
Receiver Name: Z
Transaction Value: 677
Transaction timestamp: 97
34
Sender Name: C
Receiver Name: Y
Transaction Value: 452
Transaction timestamp: 96
35
Sender Name: C
Receiver Name: Y
Transaction Value: 895
Transaction timestamp: 95
36
Sender Name: C
Receiver Name: Y
Transaction Value: 67
Transaction timestamp: 94
37
Sender Name: D
Receiver Name: X
Transaction Value: 98
Transaction timestamp: 93
38
Sender Name: A
Receiver Name: Z
Transaction Value: 204
Transaction timestamp: 92
39
Sender Name: C
Receiver Name: X
Transaction Value: 291
Transaction timestamp: 91
40
Sender Name: B
Receiver Name: Z
Transaction Value: 850
Transaction timestamp: 90
41
Sender Name: A
Receiver Name: W
Transaction Value: 912
Transaction timestamp: 89
42
Sender Name: B
Receiver Name: X
Transaction Value: 335
Transaction timestamp: 88
43
Sender Name: C
Receiver Name: Y
Transaction Value: 453
Transaction timestamp: 87
44
Sender Name: A
Receiver Name: Y
Transaction Value: 124
Transaction timestamp: 86
45
Sender Name: D
Receiver Name: Z
Transaction Value: 3
Transaction timestamp: 85
46
Sender Name: B
Receiver Name: X
Transaction Value: 641
Transaction timestamp: 84
47
Sender Name: A
Receiver Name: X
Transaction Value: 936
Transaction timestamp: 83
48
Sender Name: B
Receiver Name: W
Transaction Value: 567
Transaction timestamp: 82
49
Sender Name: B
Receiver Name: X
Transaction Value: 913
Transaction timestamp: 81
50
Sender Name: D
Receiver Name: Z
Transaction Value: 788
Transaction timestamp: 80
51
Sender Name: B
Receiver Name: Z
Transaction Value: 206
Transaction timestamp: 79
52
Sender Name: D
Receiver Name: Y
Transaction Value: 212
Transaction timestamp: 78
53
Sender Name: B
Receiver Name: W
Transaction Value: 908
Transaction timestamp: 77
54
Sender Name: A
Receiver Name: W
Transaction Value: 771
Transaction timestamp: 76
55
Sender Name: A
Receiver Name: Y
Transaction Value: 810
Transaction timestamp: 75
56
Sender Name: C
Receiver Name: W
Transaction Value: 504
Transaction timestamp: 74
57
Sender Name: C
Receiver Name: X
Transaction Value: 148
Transaction timestamp: 73
58
Sender Name: A
Receiver Name: Y
Transaction Value: 848
Transaction timestamp: 72
59
Sender Name: C
Receiver Name: Z
Transaction Value: 214
Transaction timestamp: 71
60
Sender Name: D
Receiver Name: W
Transaction Value: 774
Transaction timestamp: 70
61
Sender Name: A
Receiver Name: Z
Transaction Value: 402
Transaction timestamp: 69
62
Sender Name: A
Receiver Name: X
Transaction Value: 184
Transaction timestamp: 68
63
Sender Name: A
Receiver Name: W
Transaction Value: 989
Transaction timestamp: 67
64
Sender Name: A
Receiver Name: X
Transaction Value: 653
Transaction timestamp: 66
65
Sender Name: B
Receiver Name: X
Transaction Value: 730
Transaction timestamp: 65
66
Sender Name: C
Receiver Name: Z
Transaction Value: 718
Transaction timestamp: 64
67
Sender Name: D
Receiver Name: X
Transaction Value: 197
Transaction timestamp: 63
68
Sender Name: A
Receiver Name: Y
Transaction Value: 621
Transaction timestamp: 62
69
Sender Name: C
Receiver Name: X
Transaction Value: 598
Transaction timestamp: 61
70
Sender Name: B
Receiver Name: Z
Transaction Value: 979
Transaction timestamp: 60
71
Sender Name: A
Receiver Name: W
Transaction Value: 1000
Transaction timestamp: 59
72
Sender Name: D
Receiver Name: Z
Transaction Value: 793
Transaction timestamp: 58
73
Sender Name: D
Receiver Name: Z
Transaction Value: 530
Transaction timestamp: 57
74
Sender Name: B
Receiver Name: Z
Transaction Value: 868
Transaction timestamp: 56
75
Sender Name: C
Receiver Name: Y
Transaction Value: 774
Transaction timestamp: 55
76
Sender Name: D
Receiver Name: Y
Transaction Value: 205
Transaction timestamp: 54
77
Sender Name: A
Receiver Name: Z
Transaction Value: 834
Transaction timestamp: 53
78
Sender Name: A
Receiver Name: Y
Transaction Value: 263
Transaction timestamp: 52
79
Sender Name: A
Receiver Name: Y
Transaction Value: 306
Transaction timestamp: 51
80
Sender Name: A
Receiver Name: X
Transaction Value: 473
Transaction timestamp: 50
81
Sender Name: B
Receiver Name: W
Transaction Value: 287
Transaction timestamp: 49
82
Sender Name: A
Receiver Name: Y
Transaction Value: 43
Transaction timestamp: 48
83
Sender Name: B
Receiver Name: W
Transaction Value: 137
Transaction timestamp: 47
84
Sender Name: A
Receiver Name: Y
Transaction Value: 943
Transaction timestamp: 46
85
Sender Name: D
Receiver Name: W
Transaction Value: 479
Transaction timestamp: 45
86
Sender Name: C
Receiver Name: W
Transaction Value: 956
Transaction timestamp: 44
87
Sender Name: D
Receiver Name: X
Transaction Value: 36
Transaction timestamp: 43
88
Sender Name: B
Receiver Name: X
Transaction Value: 560
Transaction timestamp: 42
89
Sender Name: A
Receiver Name: W
Transaction Value: 663
Transaction timestamp: 41
90
Sender Name: B
Receiver Name: W
Transaction Value: 365
Transaction timestamp: 40
91
Sender Name: D
Receiver Name: X
Transaction Value: 950
Transaction timestamp: 39
92
Sender Name: B
Receiver Name: X
Transaction Value: 959
Transaction timestamp: 38
93
Sender Name: B
Receiver Name: Z
Transaction Value: 869
Transaction timestamp: 37
94
Sender Name: D
Receiver Name: Y
Transaction Value: 621
Transaction timestamp: 36
95
Sender Name: A
Receiver Name: X
Transaction Value: 881
Transaction timestamp: 35
96
Sender Name: A
Receiver Name: X
Transaction Value: 405
Transaction timestamp: 34
97
Sender Name: A
Receiver Name: Y
Transaction Value: 812
Transaction timestamp: 33
98
Sender Name: B
Receiver Name: W
Transaction Value: 68
Transaction timestamp: 32
99
Sender Name: D
Receiver Name: W
Transaction Value: 728
Transaction timestamp: 31
100
Sender Name: C
Receiver Name: W
Transaction Value: 349
Transaction timestamp: 30
101
Sender Name: A
Receiver Name: Z
Transaction Value: 935
Transaction timestamp: 29
102
Sender Name: D
Receiver Name: Y
Transaction Value: 623
Transaction timestamp: 28
103
Sender Name: B
Receiver Name: Y
Transaction Value: 926
Transaction timestamp: 27
104
Sender Name: D
Receiver Name: Z
Transaction Value: 259
Transaction timestamp: 26
105
Sender Name: D
Receiver Name: X
Transaction Value: 618
Transaction timestamp: 25
106
Sender Name: B
Receiver Name: Y
Transaction Value: 634
Transaction timestamp: 24
107
Sender Name: D
Receiver Name: Z
Transaction Value: 517
Transaction timestamp: 23
108
Sender Name: B
Receiver Name: W
Transaction Value: 672
Transaction timestamp: 22
109
Sender Name: C
Receiver Name: Y
Transaction Value: 346
Transaction timestamp: 21
110
Sender Name: C
Receiver Name: Z
Transaction Value: 400
Transaction timestamp: 20
111
Sender Name: B
Receiver Name: W
Transaction Value: 770
Transaction timestamp: 19
112
Sender Name: A
Receiver Name: Y
Transaction Value: 723
Transaction timestamp: 18
113
Sender Name: A
Receiver Name: X
Transaction Value: 467
Transaction timestamp: 17
114
Sender Name: B
Receiver Name: W
Transaction Value: 2
Transaction timestamp: 16
115
Sender Name: D
Receiver Name: Y
Transaction Value: 702
Transaction timestamp: 15
116
Sender Name: A
Receiver Name: Z
Transaction Value: 83
Transaction timestamp: 14
117
Sender Name: B
Receiver Name: X
Transaction Value: 92
Transaction timestamp: 13
118
Sender Name: D
Receiver Name: Y
Transaction Value: 195
Transaction timestamp: 12
119
Sender Name: C
Receiver Name: Z
Transaction Value: 37
Transaction timestamp: 11
120
Sender Name: A
Receiver Name: Z
Transaction Value: 756
Transaction timestamp: 10
121
Sender Name: C
Receiver Name: Y
Transaction Value: 300
Transaction timestamp: 9
122
Sender Name: A
Receiver Name: Y
Transaction Value: 671
Transaction timestamp: 8
123
Sender Name: D
Receiver Name: Z
Transaction Value: 278
Transaction timestamp: 7
124
Sender Name: B
Receiver Name: Z
Transaction Value: 687
Transaction timestamp: 6
125
Sender Name: A
Receiver Name: Y
Transaction Value: 566
Transaction timestamp: 5
126
Sender Name: D
Receiver Name: Y
Transaction Value: 936
Transaction timestamp: 4
127
Sender Name: B
Receiver Name: Y
Transaction Value: 927
Transaction timestamp: 3
128
Sender Name: B
Receiver Name: Y
Transaction Value: 676
Transaction timestamp: 2
129
Sender Name: D
Receiver Name: W
Transaction Value: 201
Transaction timestamp: 1